code/game/machinery/computer/crew.dm

```diff
@@ -150,7 +150,7 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 		"Emergency Response Team Commander" = 220,
 		"Security Response Officer" = 221,
 		"Engineer Response Officer" = 222,
-		"Medical Response Officer" = 223
```

Comment:
maybe just go off this list?

Reply:
I hate this list, ruined my whole life. I don't see why the jobs displayed on the crew monitor would be hardcoded onto it.